regression-tests.dnsdist/test_RulesActions.py

```diff
@@ -14,7 +14,7 @@ class TestAdvancedAllow(DNSDistTest):
 
     _config_template = """
     addAction(AllRule(), NoneAction())
-    addAction(SuffixMatchNodeRule("allowed.advanced.tests.powerdns.com."), AllowAction())
+    addAction(QNameSuffixRule("allowed.advanced.tests.powerdns.com."), AllowAction())
```

Comment:
you changed the name in two tests - maybe only change it in one, so we protect ourselves from accidentally removing the old name

Reply:
The old name is still used in a lot of tests! I used the new ones in two tests to prevent accidentally removing the new one :)

Reply:
oh good!